### DISCUSSION TITLE
Welcome to sciter-sdk Discussions!

### DISCUSSION THREAD

**Discussion by c-smile** [c-smile/sciter-sdk#196] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MartinN3** [comment]
Hello, im mostly UI JS developer.

Recently (for my fun raspberry pi project) i needed to interface with hardware more directly. I was searching for framework that is used to make cars UI. And while searching for alternatives i found Sciter. Seems like having everything that is needed.

If i'll be able to run sciter, i want to document my transition for other web javascript developers as an introduction.

**Reply by c-smile** [reply]
Hi Martin.

Welcome aboard.

Please use https://github.com/c-smile/sciter-js-sdk 

That is the same Sciter but with standard JavaScript.

Sciter.JS also works on Raspberry and in two models: "windowed"  Sciter and "windowless" Sciter.Lite.

Sciter.Lite is suitable for your case better as it is designed to work in so called kiosk mode (window spans whole monitor).
Yet Sciter.Lite supports as bitmap rendering targets (like DirectFB) as H/W accelerated OpenGL or Vulkan (next releases).
